python/ribasim/ribasim/input_base.py

```diff
@@ -392,15 +393,24 @@ def check_parent(self) -> "ChildModel":
 class NodeModel(ChildModel):
     """Base class to handle combining the tables for a single node type."""
 
-    _sort_keys: dict[str, list[str]] = {}
-
     @model_serializer(mode="wrap")
     def set_modeld(
         self, serializer: Callable[[type["NodeModel"]], dict[str, Any]]
     ) -> dict[str, Any]:
         content = serializer(self)
         return dict(filter(lambda x: x[1], content.items()))
 
+    @field_validator("*")
+    @classmethod
+    def set_sort_keys(cls, v: Any, info: ValidationInfo) -> Any:
+        """Set sort keys for all TableModels if present in FieldInfo."""
+        if isinstance(v, (TableModel,)):
+            field = cls.model_fields[getattr(info, "field_name")]
+            extra = field.json_schema_extra
+            if extra is not None and isinstance(extra, dict):
+                v.sort_keys = extra.get("sort_keys", [])  # type: ignore
```

Comment:
Could you add the reason why you added a type ignore? Is it something we want to fix?

Reply:
I updated the code, and provided an explanation. Not something we want to fix imho.